src/components/PreviewComment/index.js

```diff
@@ -108,7 +108,7 @@ export const styles = StyleSheet.create({
   },
   container: (totalComment) => ({
     borderLeftWidth: 1,
-    marginHorizontal: SIZES.base,
+    marginHorizontal: SIZES.base - 1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in `marginHorizontal` from `SIZES.base` to `SIZES.base - 1` might cause layout issues if not properly tested on different screen sizes and resolutions. Ensure that this change doesn't affect the responsiveness of the UI across different devices.
```diff
-    marginHorizontal: SIZES.base - 1,
+    marginHorizontal: SIZES.base,
```
Please verify this change with thorough testing or revert it back to the original value if it's not necessary.


<!-- This is an auto-generated comment by OSS CodeRabbit -->